sources/web/datalab/polymer/components/file-browser/file-browser.ts

```diff
@@ -142,6 +137,17 @@ class FileBrowserElement extends Polymer.Element {
 
     super.ready();
 
+    this.$.breadCrumbs.addEventListener('crumbClicked', (e: CustomEvent) => {
```

Comment:
In bread-crumbs.ts you dispatch an ItemClickedEvent, should that be the type of e here?